### PR TITLE
Add global search to template

### PR DESCRIPTION
**Pull request by tmotyl**
see https://github.com/TYPO3-Documentation/t3docs-search-indexer
https://docs.typo3.org/search/

Make global search input box appear on the right side from the logo.

### CODE REVIEW

**Comment by tmotyl** [comment]
I see that we would also need to adapt the 404 page to mention the new search. @marble can you point me to where is the template of the 404 page located?

**Comment by susannemoog** [comment]
@tmotyl this one? https://github.com/TYPO3-Documentation/DocsTypo3Org-Homepage/blob/master/Documentation/404.rst ? (click "edit on github" on the 404 :))

**Comment by tmotyl** [comment]
> @tmotyl this one? https://github.com/TYPO3-Documentation/DocsTypo3Org-Homepage/blob/master/Documentation/404.rst ? (click "edit on github" on the 404 :))

oh, right, that was too easy, I was looking for it in wrong repo :)

File: sphinx_typo3_theme/layout.html [hunk 1]
```diff
@@ -112,13 +112,15 @@
             <div class="page-header-inner">
 <!-- pageheader -->
 {%- block pageheader %}
+<div class="row">
+<div class="col-sm-6 col-lg-7">
```
Comment (simonschaufi) [inline sphinx_typo3_theme/layout.html:116]: please use correct indentation

**Comment by marble** [comment]
Done!
Has been cherry picked as https://github.com/TYPO3-Documentation/sphinx_typo3_theme/commit/a577246b22d656485d1209bba797f1161bef24a3

**Pull request closed without merging**